mkdocs.yml

```
@@ -46,8 +46,9 @@ nav:
           - Document a sample: contribute/sample_documentation.md
           - Contributors: contribute/contributors.md
     - Services:
-          - Available services: services.md
+        - Available services: services.md
```

You propably want to re add this indentation because currently this is not building.
```suggestion
          - Available services: services.md
```